### PR TITLE
adde some extra bugs to you as gifrs

### PR DESCRIPTION
**Pull request by Naaru-01**

### CODE REVIEW

**Comment by Naaru-01** [comment]
ab to kya hi comment kar sakate hai bhai

File: README.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+<<<<<<< HEAD
+hiiiiiiiiiii
```
Comment (NC24Technologies) [inline README.md]: code should be fixed from this line

File: README.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
+=======
+added the variables and made the code chagnes in the function name
+
+
+>>>>>>> b48722ca723f7f2a4bea7639e5919b7a8bfbd123
+
```
Comment (NC24Technologies) [inline README.md]: fixing of code should be end here

**Review by NC24Technologies — COMMENTED** [review]
reviewed the file

**Comment by NC24Technologies** [comment]
Thanks ! We appreciate your contribution to our organization

**Pull request merged**